ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/explorer/project/ProjectExplorerViewImpl.java

```diff
@@ -126,66 +91,12 @@ public int compare(Node o1, Node o2) {
             }
         }, SortDir.ASC));
 
-        if (tree.getGoInto() != null) {
```

Comment:
We don't have following method anymore.
```
public void onGoIntoStateChanged(GoIntoStateEvent event)
```
Now we have action in part menu for go into.